src/Services/Kros.Authorization.Api/Application/Services/IUserService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Kros.Authorization.Api.Application.Commands;
+using Kros.Authorization.Api.Application.Model;
+using Kros.Authorization.Api.Application.Queries;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Kros.Authorization.Api.Application.Services
+{
+    /// <summary>
+    /// Interface which describe service for working with <see cref="User"/>.
+    /// </summary>
+    public interface IUserService
+    {
+        /// <summary>
+        /// Is user admin?
+        /// </summary>
+        /// <param name="user">User for detect.</param>
+        /// <returns>true, if it's admin, false otherwise.</returns>
```

Comment:
Use `<see langword="true"/>`. It will be better visualised in intelliSense and in the generated documentation.